src/components/CreateTarget/CreateTarget.js

```diff
@@ -10,10 +10,9 @@ import target from 'assets/icons/target.svg';
 import 'components/CreateTarget/CreateTarget.scss';
 
 const CreateTarget = ({ onContinue }) => {
-  const { selectedLocation } = useContext(MapContext);
+  const { mapProperties, setMapProperties } = useContext(MapContext);
```

Comment:
you could destructure this too 
```suggestion
  const { mapProperties: { selectedRadius, ... }, setMapProperties } = useContext(MapContext);
```